### DISCUSSION TITLE
Error while trying to install on an angular project

### DISCUSSION THREAD

**Discussion by jurgenru** [abacritt/angularx-social-login#471] [Q&A] [answered]
![Screenshot from 2022-01-15 23-40-32](https://user-images.githubusercontent.com/37807383/149646354-15e589cd-d9fd-4e7d-9724-1d0b47000c46.png)
I get this error everytime i try to install it

**Comment by ranavj** [comment]
same error i am getting

**Reply by DaniVagyok** [reply]
anything??

**Comment by juanmaldonadodev** [comment]
> ![Screenshot from 2022-01-15 23-40-32](https://user-images.githubusercontent.com/37807383/149646354-15e589cd-d9fd-4e7d-9724-1d0b47000c46.png) I get this error everytime i try to install it

The library is using angular 12 while your project is using angular 13 you have a conflict there

**Comment by Heatmanofurioso** [comment, marked as answer]
Hi @jurgenru 
If you use the latest version of the library, it does use Angular 13.

Also note that the library is now at `@abacritt/angularx-social-login`, and not in `angularx-social-login`

So, all the latest versions & updates are on that NPM namespace 

See https://www.npmjs.com/package/@abacritt/angularx-social-login
